src/terraform_drift_detection/detect.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import logging
+
+from terraform_drift_detection.init import init
+from terraform_drift_detection.terraform import check_repos
+from terraform_drift_detection.util import Drift
+
+init()
+if Drift.DRIFT in check_repos():
+  logging.error('Drift detected. Please see logs.')
+  exit(1)
```

Comment:
Non-zero exit causes k8s to consider the job failed. The are 3 possible situations:
- Terraform commands (init/plan) work. Result is no drift. Script exits 0.
- Commands work. Result is drift. Script exits 1.
- Commands fail (git clone problem, dynamo db terraform table has a lock entry, ...), Script exits 0. I did consider exiting 1 (to alert us), but that seems outside the scope of this job, as its purpose is to detect drift, not terraform problems.